en: add tune-configuration.md and update dm-upgrade.md (#130)
pingcap/docs-dm: en/dm-upgrade.md

> - Unless otherwise stated, all the following upgrade examples assume that all the data replication tasks have been stopped before the upgrade and all the replication tasks are restarted manually after DM upgrade is finished.
> - The following shows the upgrade procedure of DM versions in reverse chronological order.

## Upgrade to v1.0.5

### Version information

```bash
Release Version: v1.0.5
Git Commit Hash: a8e9f53f91e29756b09a22cdc37a6a6efcdfe55b
Git Branch: release-1.0
UTC Build Time: 2020-04-27 06:56:31
Go Version: go version go1.13 linux/amd64
```

### Main changes

> - Optimized the incremental synchronization speed when the column corresponding to `UNIQUE KEY` contains `NULL` value

Reviewer: It is recommended to use [present tense](https://developers.google.com/style/tense)

- Added retry for the error of `Write conflict` (9007 and 8005) returned by TiDB
- Fixed the issue that may cause `Duplicate entry` errors during the full data import process

> - Fixed the issue where `stop-task`/`pause-task` may not work when there is no data written upstream after the full import is completed

Reviewer: ```suggestion
- Fix the issue that the `stop-task`/`pause-task` command may not work when no data written upstream after the full import is completed
```

> - Fixen the issue that there is still data display in the monitoring metrics after `stop-task`

Reviewer: ```suggestion
- Fix the issue the monitoring metrics still display data after the replication task is stopped
```

### Upgrade operation example

1. Download the new version of DM-Ansible, and confirm that there is `dm_version = v1.0.5` in the `inventory.ini` file.
2. Run `ansible-playbook local_prepare.yml` to download the new DM binary file to the local disk.
3. Run `ansible-playbook rolling_update.yml` to perform a rolling update for the DM cluster components.
4. Run `ansible-playbook rolling_update_monitor.yml` to perform a rolling update for the DM monitoring components.

## Upgrade to v1.0.4

### Version information

```bash
Release Version: v1.0.4-1-gd681c67
Git Commit Hash: d681c6731d3432f4d8f38ea651f44d49d6860269
Git Branch: release-1.0
UTC Build Time: 2020-03-16 09:45:29
Go Version: go version go1.13 linux/amd64
```

### Main changes

> - Add support for English UI in DM Portal

Reviewer: ```suggestion
- Add English UI for DM Portal
```

> - Added support for English UI in DM Portal

Reviewer: Please change `past tense` to `present tense`, `Fixed the issue where` to `Fix the issue that`~

> - Add `--more` parameter to `query-status` command to display complete synchronization status information

Reviewer: ```suggestion
- Add the `--more` parameter in the `query-status` command to show complete replication status information
```

> - Fix the issue that the resume-task may not be able to resume synchronization normally after synchronization to the downstream TiDB connection is abnormal

Reviewer: ```suggestion
- Fix the issue that `resume-task` might fail to resume the replication task which is interrupted by the abnormal connection to the downstream TiDB server
```

> - Fix the issue that the online DDL meta information was cleaned up incorrectly after the online DDL failed to execute, and the online DDL synchronization cannot be processed correctly after restarting the task

Reviewer: ```suggestion
- Fix the issue that the online DDL operation cannot be properly replicated after a failed replication task is restarted because the online DDL meta information has been cleared after the DDL operation failure
```

> - Fix the issue that `start-task` abnormally returned `query-error` may cause DM-worker panic

Reviewer: ```suggestion
- Fix the issue that `query-error` might cause the DM-worker to panic after `start-task` goes into error
```

> - Fix the issue that the DM-worker process stopped abnormally before writing to `relay.meta`, resulting in the problem that the relay log file and `relay.meta` may not be restored correctly when restarting the DM-worker

Reviewer: ```suggestion
- Fix the issue that the relay log file and `relay.meta` cannot be correctly recovered when restarting an abnormally stopped DM-worker process before `relay.meta` is successfully written
```
